src/web/routes/application/children-dob/decrement-keys.test.js

```diff
@@ -0,0 +1,34 @@
+const test = require('tape')
+const { isKeyAfterRemoved, decrementKey } = require('./decrement-keys')
+
+test('isKeyAfterRemoved()', (t) => {
+  t.equal(isKeyAfterRemoved('childDob-1-day', 1), false)
+  t.equal(isKeyAfterRemoved('childDob-1-day', 2), false)
+  t.equal(isKeyAfterRemoved('childDob-1-day', 3), false)
+
+  t.equal(isKeyAfterRemoved('childDob-2-day', 1), true)
+  t.equal(isKeyAfterRemoved('childDob-2-day', 2), false)
+  t.equal(isKeyAfterRemoved('childDob-2-day', 3), false)
+
+  t.equal(isKeyAfterRemoved('childDob-3-day', 1), true)
+  t.equal(isKeyAfterRemoved('childDob-3-day', 2), true)
+  t.equal(isKeyAfterRemoved('childDob-3-day', 3), false)
+
+  t.equal(isKeyAfterRemoved('childDob-11-day', 1), true)
+  t.equal(isKeyAfterRemoved('childDob-1', 1), false)
+  t.equal(isKeyAfterRemoved('childDob-5', 1), true)
+
+  t.throws(() => isKeyAfterRemoved('childDob-day', 1), /Could not find index for key childDob-day/)
+  t.throws(() => isKeyAfterRemoved('inputCount', 1), /Could not find index for key inputCount/)
+  t.end()
+})
+
+test('decrementKey()', (t) => {
+  t.equal(decrementKey('childDob-2-day'), 'childDob-1-day')
+  t.equal(decrementKey('childDob-11-day'), 'childDob-10-day')
```

Comment:
Might be worth adding a test for 'childDob-10-day' to confirm that the change from 2 digits to 1 is handled correctly